programs/intent-transfer/src/bridge/processor/bridge_ntt_tokens.rs

```diff
@@ -20,15 +20,10 @@ use solana_intents::Intent;
 const BRIDGE_NTT_INTERMEDIATE_SEED: &[u8] = b"bridge_ntt_intermediate";
 const BRIDGE_NTT_NONCE_SEED: &[u8] = b"bridge_ntt_nonce";
 
-// TODO: we should do some parsing of the relay_instructions and/or exec_amounts arg(s)
-// in order to ensure the signed intent message does precisely what the user expects
-// this will help to prevent MITM attacks
 #[derive(AnchorSerialize, AnchorDeserialize, Clone)]
 pub struct BridgeNttTokensArgs {
-    // TODO: we need to safely ensure the exec_amount cannot be spoofed
-    pub exec_amount: u64,
     pub signed_quote_bytes: Vec<u8>,
-    pub relay_instructions: Vec<u8>,
+    pub pay_destination_rent: bool,
```

Comment:
nit: executor is already always paying rent for wormhole accounts, difference is whether it pays for the recipient ata rent or not